source/languages/en/dataplatform/using-dataplatform/configuration/replace-spark-cluster-manager.md

```diff
@@ -40,18 +40,18 @@ To replace your Spark Cluster Manager with the BDP cluster manager, you will do
 
 1. Add BDP service configs for spark-master and spark-worker.
 2. Activate the services.
-3. Verify
+3. Verify your success.
```

Comment:
Here as well, @ph07 , would you like to update this PR, or should I make the change myself?